django_password_validators/password_history/password_validation.py

```diff
@@ -12,11 +12,22 @@
 
 class UniquePasswordsValidator(object):
     """
-    Validate whether the password was once used by the user.
-
+    Validate whether the password was once used by the user
+    in a determined range.
     The password is only checked for an existing user.
     """
 
+    def __init__(self, lookup_range=float('inf')):
+        try:
+            self.lookup_range = lookup_range if lookup_range >= 0 else float(
+                'inf')
```

Comment:
@spokerman12, the line break at this place makes the code slightly less readable. The below version is more readable, no?

```suggestion
            self.lookup_range = (
                lookup_range if lookup_range >= 0 else float(inf)
            )
```